ChangeLog

```diff
@@ -1,6 +1,8 @@
 2025-01-01  Bill Denney  <wdenney@humanpredictions.com>
 
-  * R/sha1.R: Default sha1 method now works for all classes by default
+	* R/sha1.R: Default sha1 method now works for all classes by default. If you
```

Comment:
s/all/more/ ?  Do we know it's all?

I'd remove the "If you ..." letter to the reader.  Not what we commonly put in ChangeLog :)  Minor style issue.